endoreg_db/views/video_anonymization_views.py

```
@@ -0,0 +1,460 @@
+    
+    Returns:
+    - success: bool
+    - message: str
+    - video_id: str
+    - anonymized: bool
+    """
+    try:
+        video = get_object_or_404(VideoFile, uuid=video_id)
+        
+        # Parse request parameters
+        delete_original_raw = request.data.get('delete_original_raw', True)
```

Ruff (0.11.9)

Local variable `delete_original_raw` is assigned to but never used

Remove assignment to unused variable `delete_original_raw`

(F841)